### PR TITLE
Clarify App Engine SDK selection instructions in Development Guidelines #6709

### LINKED ISSUE #6709: Clarify App Engine SDK selection instructions in Development Guidelines
Line 48 (shown below) of `development.md` is unclear as to which `appengine-sdk` folder the user is supposed to select (`appengine-sdk` or `appengine-java-sdk-<version>`).  This needs to be clarified.
> **Line 48**
> `Click + → .... Select the App Engine SDK you downloaded in Step 3 of the Setting up a development environment guide.`

From @damithc:
> If the user chose the appengine-sdk folder instead of the sub-folder inside it (note that `appengine-sdk` is the folder mentioned in Step 3) it will result in this error at a later stage:
`Error running Dev Server: 'C:/Users/damithch/.gradle/appengine-sdk' isn't valid App Engine SDK installation: 'C:\Users\damithch\.gradle\appengine-sdk\lib\appengine-tools-api.jar' not found`

**Possible Solution:**
Give an example of the folder name to be selected in parentheses after `App Engine SDK` like so: 
> App Engine SDK (`appengine-java-sdk-<version>`)

Other suggestions are welcome (you can post your suggestion as a comment here first).

### PR DESCRIPTION
**Pull request by Skchoudhary**
Clarify App Engine SDK selection instructions in Development Guidelines #6709

Fixes #6709

### CODE REVIEW

**Comment by samsontmr** [comment]
@Skchoudhary good job on making your first PR! Note that in the future, your PR title should be `<exact copy of issue title>-#<issue-number>`.

File: docs/development.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Click the "Terminate" icon on the Eclipse console.
 1. Click `+ → Google AppEngine Dev Server`.
 1. Name it `Dev Server`.
 1. Click `Configure` next to `Application server`.
-1. Click `+ → ...`. Select the App Engine SDK you downloaded in Step 3 of the [Setting up a development environment](settingUp.md) guide.
+1. Click `+ → ...`. Select the App Engine SDK (appengine-java-sdk-<version> sub-folder) you downloaded in Step 3 of the [Setting up a development environment](settingUp.md) guide.
```
Comment (damithc) [inline docs/development.md]: Format as follows?
`appengine-java-sdk-<version>` sub-folder

Comment (Skchoudhary) [inline docs/development.md]: So 
it should look like something this way
`Click `+ → ...`. Select the App Engine SDK→ appengine-java-sdk-<version> sub-folder you downloaded in Step 3 of the [Setting up a development environment](settingUp.md) guide.`

Comment (damithc) [inline docs/development.md]: No. Like this:

1. Click `+ → ...`. Select the App Engine SDK (`appengine-java-sdk-<version>` sub-folder) you downloaded in Step 3 of the [Setting up a development environment](settingUp.md) guide.

Comment (samsontmr) [inline docs/development.md]: @Skchoudhary surround only `appengine-java-sdk-<version>` with backticks.

Comment (Skchoudhary) [inline docs/development.md]: ok

**Pull request merged**